cron/bsc/standalone-runner.js

```diff
@@ -13,6 +13,76 @@ const vaultDecision = require('./vault-decision.json')
 
 const ethers = hre.ethers
 
+// Prometheus monitoring
+const promClient = require('prom-client')
+const Registry = promClient.Registry
+const register = new Registry()
+
+async function pushMetrics(labels) {
+  const gateway = new promClient.Pushgateway(
+    settings.prometheusMonitoring.pushGatewayUrl,
+    [],
+    register,
+  )
+  return gateway
+    .push(labels)
+    .then(({ resp, body }) => {
+      console.log(`Metrics pushed, status ${resp.statusCode} ${body}`)
+      register.clear()
+    })
+    .catch(err => {
+      console.log(`Error pushing metrics: ${err}`)
+    })
+}
+
+async function reportSimulationProfit(vault, block, ethProfit, execute) {
+  const profitMetric = new promClient.Gauge({
+    name: 'eth_profit',
+    help: 'profit shared Ether',
+    registers: [register],
+  })
+  register.registerMetric(profitMetric)
```

Comment:
just checking: if metric is already registered, it is okay?

Reply:
and we do not require `await`, is it right?

Reply:
shouldn't be a problem